packages/react-storage/src/components/StorageBrowser/createStorageBrowser/types.ts

```diff
@@ -306,20 +315,22 @@ export type DerivedActionViewType<T extends StorageBrowserActions> =
  * @description return values of `createStorageBrowser`
  */
 export interface CreateStorageBrowserOutput<
-  C extends ExtendedActionConfigs = ExtendedActionConfigs,
+  ActionsType extends StorageBrowserActions,
```

Comment:
`ActionsType` reads like a union of `Actions`. Prefer generic names aligning to the template it extends from, example:
```ts
export interface CreateStorageBrowserOutput<
  TActions extends StorageBrowserActions,
> {
```